lib/dialects/abstract/query-generator.js

```diff
@@ -774,7 +774,10 @@ const QueryGenerator = {
     Currently this function is only used for ordering / grouping columns and Sequelize.col(), but it could
     potentially also be used for other places where we want to be able to call SQL functions (e.g. as default values)
   */
-  quote(obj, parent, force) {
+  quote(obj, parent, force, connector) {
+
+    connector = connector === undefined ? '.' : connector;
```

Comment:
After getting burned on values that resolve to false but are valid I tend to be paranoid about just saying `!var`, but I guess it is pretty safe in this case. I will go ahead and use `connector || '.'`